src/app/components/Modal/style.scss

```
@@ -0,0 +1,38 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  padding-top: 100px;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modalContent {
+  position: fixed;
+  left: 10%;
+  z-index: 1;
+  width: 80%;
+  margin: auto;
+  padding: 20px;
+  padding-top: 100px;
+  background-color: #fefefe;
+  border: 1px solid #888;
+}
+
+.close {
+  float: right;
+  color: #aaa;
+  font-weight: bold;
+  font-size: 28px;
```

```suggestion
  font-size: 2.8rem;
```

font-size の単位は rem で統一していますので rem で指定をお願いします
root (html) で font-size: 62.5% で規定しているので、 1rem = 10px になります。